src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const listPopulation = document.querySelectorAll('.population');
+const arrayPopulation = [...listPopulation].map(
+  item => item.innerText.replace(/,/g, '')
+);
+const numberPopulation = arrayPopulation.map(item => Number(item));
+
+const totalAmount = numberPopulation.reduce((sum, current) => sum + current);
```

Comment:
add initial value to reduce 

```suggestion
const totalAmount = numberPopulation.reduce((sum, current) => sum + current, 0);
```